### DISCUSSION TITLE
How does swap windows using mouse work?

### DISCUSSION THREAD

**Discussion by al3xandru** [ianyh/Amethyst#1619] [Q&A] [answered]
I'm sorry if the question sounds a bit naive.  I am not quite sure how swapping windows using the mouse is supposed to work.  What I'd imagine is that by dragging a window to a given area of the screen it will switch places with the app at that location.  I don't think I've seen this working though.

**Comment by niemes** [comment, marked as answer]
Hi, 

It's drag and drop, you drag one window using the title bar to another window and they swap.
I use it all the time, works well.

**Reply by ianyh** [reply]
@al3xandru if this is not working for you, you should open an Issue so that I can take a look at your particular case.
